app/models/ticket.rb

```diff
@@ -5,11 +5,16 @@ class Ticket < ApplicationRecord
   validates :entered_gate_id, presence: true
   validate :check_distance
 
+  def used?
+    exited_gate_id.present?
+  end
+
   private
 
   def check_distance
-    if exited_gate_id.present? && !(exited_gate&.exit?(self))
-      errors.add(:exited_gate_id, '降車駅 では降車できません。')
+    if exited_gate_id.present? && !exited_gate&.exit?(self)
```

Comment:
```suggestion
    if used? && !exited_gate&.exit?(self)
```

でもいいかな？あ、あとusedが確定するなら`&.`は不要なのでは？

Reply:
修正します！